build/yamls/flow-aggregator/base/conf/flow-aggregator.conf

```diff
@@ -2,11 +2,20 @@
 # If no L4 transport proto is given, we consider tcp as default.
 #externalFlowCollectorAddr: ""
 
-# Provide flow export interval as a duration string. This determines how often the flow aggregator exports flow
-# records to the flow collector.
-# Flow export interval should be greater than or equal to 1s (one second).
+# Provide the active flow record timeout as a duration string. This determines
+# how often the flow aggregator exports the active flow records to the flow
+# collector. Thus, for flows with a continuous stream of packets, a flow record
+# will be exported to the collector once the elapsed time since the last export
+# event is equal to the value of this timeout.
 # Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
-#flowExportInterval: 60s
+#activeFlowRecordTimeout: 60s
+
+# Provide the inactive flow record timeout as a duration string. This determines
+# how often the flow aggregator exports the inactive flow records to the flow
+# collector. A flow record is considered to be inactive if no matching record
+# has been received from any Node in the cluster since the last export event.
+# Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
+#inactiveFlowRecordTimeout: 90s
```

Comment:
following our offline discussion the other day, I think it would be good as a follow up PR to update the user-facing documentation with information on how to choose these values for the best results (in particular how they should be ordered and why)